tabitha-json/build.gradle

```diff
@@ -0,0 +1,6 @@
+description = 'Tabitha support for delimiter-based formats'
```

Comment:
Copy-pasta that has yet to be changed? :)

Reply:
Indeed.